website/docs/failing-closed.md

```diff
@@ -0,0 +1,148 @@
+---
+id: failing-closed
+title: Failing Closed
+---
+
+Here we discuss how to configure Gatekeeper to fail closed and some factors you may want to consider before doing so.
+
+## How to Fail Closed
+
+If you installed Gatekeeper via the manifest, the only needed change is to set Gatekeeper's `ValidatingWebhookConfiguration` to fail. For example:
```

Comment:
Should we mention `failurePolicy` here? it's unclear what changed in vwc below

Reply:
Good catch, adding this as well.